tests/v2/validation/provisioning/rke2/README.md

```diff
@@ -43,7 +43,8 @@ provisioningInput is needed to the run the RKE2 tests, specifically kubernetesVe
     "providers": ["linode", "aws", "do", "harvester"],
     "nodeProviders": ["ec2"],
     "hardened": true,
-    "psact": ""
+    "psact": "",
+    "sshTests": ["CheckCPU"]
```

Comment:
are there other options (I think no, for now)

following our last team meeting, can you also add a section on how to use sshTests, maybe even a future works bullet point asto what may be important to test later.